CS483/CS483/Kartaclysm/Components/ComponentKartController.cpp

```diff
@@ -99,6 +131,18 @@ namespace Kartaclysm
 		PlayerInputMapping::Instance()->QueryPlayerMovement(m_iPlayerNum, iAccelerate, iBrake, iSlide, fTurn);
 		fTurn *= -1.0f; // Reversed because of mismatch between what the game and the controller consider to be the positive horizontal direction
 
+		// Spinout causes all inputs to be ignored
+		if (m_fSpinout > 0.0f)
+		{
+			iAccelerate = 0;
+			iBrake = 0;
+			iSlide = 0;
+			fTurn = 0.0f;
```

Comment:
Couldn't you set these as default values and not call QueryPlayerMovements() if we're spinning out? It's a little inefficient to call it and then ignore it's results.